### DISCUSSION TITLE
When can we expect the new release?

### DISCUSSION THREAD

**Discussion by MartellX** [TNG/ArchUnit#1283] [Q&A] [answered]
Our project can't upgrade spring boot version to 3.2 because of bug https://github.com/TNG/ArchUnit/issues/1224 which was fixed but not released. 
Can we hope for a release in the near future?

**Comment by zhenya-yadlovskij** [comment]
Hi Guys! @codecholeric @hankem 
Thanks for a great product.
Could you please tell if we can expect new release which includes this fix in near future?
Or if making a release is one-press-button. Could you please release a new version?

We are interested in this issue as well so hard :(

**Reply by zhenya-yadlovskij** [reply]
Hi @hankem!
Could you please tell if we can release a new version?

**Reply by hankem** [reply]
Sorry for the delay! @codecholeric is already about to finalize the new release.

**Reply by zhenya-yadlovskij** [reply]
Awesome! Thank you guys!

**Comment by codecholeric** [comment, marked as answer]
> Our project can't upgrade spring boot version to 3.2 because of bug #1224 which was fixed but not released. Can we hope for a release in the near future?

Sorry, I thought that this was monkey patched in Moduliths! Otherwise, I would have released this fix sooner as a bugfix 🙈

**Reply by zhenya-yadlovskij** [reply]
Anyway thank you for the release!

**Reply by MartellX** [reply]
Thanks!
